content/blog/2022/2022-02-11-edit-image-collection.md

```diff
@@ -0,0 +1,91 @@
+
+[The Import Images](/widget-catalog/image-analytics/importimages/) widget loads images from the training dataset ([download the training data](https://github.com/PrimozGodec/datasets/raw/master/image-analytics-collection/train-photos.zip)). Since images are sorted in multiple folders, the widget assigns the folder name to images as a category, which we can see in the Data Table. We embed images with Inception v3 embedder in [the Image Embedding widget]((/widget-catalog/image-analytics/imageembedding/)) and performs cross-validation in [the Test and Score widget](/widget-catalog/evaluate/testandscore). The model we use is logistic regression. We are happy with the model with AUC 0.992 and classification accuracy of 0.943.
+
+Now we can train the model that will be used to label an unlabeled set of photos.
+
+{{< window-screenshot src="/blog_img/2022/2022-02-11-image-classification.jpg" >}}
+
+We train [the logistic regression model](/widget-catalog/model/logisticregression) on the training images in the top branch on the labelled training dataset. Data that we want to label (sort in subfolders) are loaded and embedded in the bottom branch. We use [the same images](https://github.com/PrimozGodec/datasets/blob/master/image-analytics-collection/test-photos.zip) as in the first part of the blog. [The Predictions widget](/widget-catalog/evaluate/predictions) assigns labels to the images based on the logistic regression model.
+
+We use [the Image Viewer](/widget-catalog/image-analytics/imageviewer) to inspect assigned labels to images. Labels seem to be meaningful and correct. Now that we have photos labelled, we save them to the folder structure with [the Save Images widget](/widget-catalog/image-analytics/saveimages) such that each folder is one category. Since the Prediction widget places predictions (column named Logistic Regression) in the meta part of the table, we need to first set the attribute as a target with the Select Columns widget --- the Save Images widget uses the target column as a subfolder name.
+
+We also connected [the Save Model widget](/widget-catalog/model/savemodel) to the Logistic Regression. This way, the model is saved for later and can be reused. 
+
+### Want to learn more?
+
+We showed two ways of analysing images with Orange's Image Analytics module. The Image Grid is a great tool to identify similar images and see them in the grid. The second one needs an extra effort to prepare a training dataset, but this leads to an automatic pipeline to identify categories.
+
+If you want to learn more about Image Analysis in [Orange watch the tutorial wideos](https://youtube.com/playlist?list=PLmNPvQr9Tf-aRstY69vGAPO_c5RaceBhN) or read related blogs: [Image Analytics: Clustering](/blog/2017/04/03/image-analytics-clustering/]), [Clustering of Monet and Manet](/blog/2018/05/09/clustering-of-monet-and-manet/), and [Outliers in Traffic Signs](/blog/2017/04/25/outliers-in-traffic-signs/)
```

Comment:
[Orange watch the tutorial wideos] --> Orange [watch the tutorial videos]
(Orange should probably not be included in the link?)